src/journal/views.py

```diff
@@ -76,6 +76,11 @@ def single_issue(request, issue_number):
     cover = Cover.objects.filter(issue=requested_issue).first()
     story_set = requested_issue.get_story_set().order_by('number')
 
+    height_of_bandcamp_plugin = 120
```

Comment:
We should be able to do this in the templates -- we're passing `story_set` as part of the context anyway, so the markup and styling logic can be worked out in jinja.

Ideally we'd have logic that assigns html attributes (like `id` or `class`), and then css/sass that actually applies height values.

Ping me if that doesn't make sense.